examples/crab-saber/src/lib.rs

```diff
@@ -177,57 +177,54 @@ fn handle_state_change(
     game_context: &mut GameContext,
     world: &mut World,
 ) {
-    let mut objects_to_hide = Vec::new();
-    let mut objects_to_show = Vec::new();
-
     match (previous_state, current_state) {
         (SessionState::VISIBLE, SessionState::FOCUSED) => {
+            fn show(world: &mut World, entity: Entity) {
```

Comment:
Do you think you could define the function outside the match statement? Might be a bit cleaner!

Reply:
Sure, done.